app/windows/runner_panel.py

```diff
@@ -404,3 +404,10 @@ def finished(self) -> None:
 
         # Cleanup process
         self.process.terminate()
+        answer = show_dialogue_conditional(
```

Comment:
Just use a BinaryChoiceDialog or show_dialogue_conditional, omit the information part, and label the two buttons in a clear manner like the examples I gave before.

Reply:
done